docs/install.md

```diff
@@ -363,6 +363,10 @@ data:
 
 To customize the behavior of HA for the Tekton Pipelines controller, please refer to the related [documentation](developers/enabling-ha.md).
 
+## Overwriting the ThreadsPerController, QPS and Burst
+
+If you want to modify the default ThreadsPerController, QPS and Burst to meet your performance requirements, then please refer to this [guidance](developers/configuring-qps-burst-thread.md).
```

Comment:
It might be worth adding a little bit of context here. Something like this:

> Out-of-the-box, Tekton Pipelines is configured for relatively small-scale deployments but several options for configuring Pipelines' performance are available. See the [Performance Configuration](./performance-configuration.md) document which describes how to change the `ThreadsPerController`, `QPS` and `Burst` settings to meet your requirements.

Reply:
Done!